samples/bluetooth/fast_pair/locator_tag/sysbuild/ipc_radio/prj.conf

```diff
@@ -25,6 +24,3 @@ CONFIG_BT_BUF_ACL_TX_SIZE=87
 CONFIG_BT_BUF_ACL_RX_SIZE=87
 CONFIG_BT_CTLR_DATA_LENGTH_MAX=87
 CONFIG_BT_CTLR_TX_PWR_DYNAMIC_CONTROL=y
-
-CONFIG_IPC_RADIO_BT=y
-CONFIG_IPC_RADIO_BT_HCI_IPC=y
```

Comment:
Can we apply similar "cleanup" for our other applications (may be done under a follow-up ticket)?